alchemist/alchemist-swingui/src/main/java/it/unibo/alchemist/boundary/gui/controller/ButtonsBarController.java

```diff
@@ -33,6 +34,7 @@
  *
  * @param <P> the position type
  */
+@SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH", justification = "Using assert to null-check avoids the possibility of null references")
```

Comment:
No, it does not. `assert` in Java are generally disabled at runtime, and only enabled with -ag. Remove this suppression, and use `Objects.requireNonNull`